src/talos/moi_tests.py

```diff
@@ -673,6 +674,104 @@ def run(
         return classifications
 
 
+class XDominantFemaleInactivation(BaseMoi):
```

Comment:
I am trying to think of the best class name for this. `XPseudoDominantFemale` is the best I can come up with.

```suggestion
class XPseudoDominantFemale(BaseMoi):
```